lua/conform/runner.lua

```diff
@@ -298,6 +298,10 @@ local last_run_errored = {}
 ---@param callback fun(err?: conform.Error, output?: string[])
 ---@return integer? job_id
 local function run_formatter(bufnr, formatter, config, ctx, input_lines, opts, callback)
+  vim.api.nvim_exec_autocmds("User", {
+    pattern = "ConformPreFormatter",
+    data = { formatter = formatter },
```

Comment:
The whole formatter contains a lot of fields that probably aren't very useful. Could we start with just passing the name and then add other data if it's needed?

Reply:
Good point! I've left the structure of `data.formatter.name` so it can be easily extended in the future